### DISCUSSION TITLE
Support for OAuth2.0 planned?

### DISCUSSION THREAD

**Discussion by Dman63** [usebruno/bruno#341] [Q&A]
Hi, 

any plans to add OAuth2.0 as an authorization method?
I can use the bearer Token, but OAuth2.0 support would be great.

**Comment by rmueller83** [comment]
See https://github.com/usebruno/bruno/issues/119

**Comment by helloanoop** [comment]
You can see our roadmap here: https://github.com/usebruno/bruno/discussions/384

**Reply by Dman63** [reply]
thanks, I'll keep an eye on it and hope it is added soon.

**Comment by tsteckenborn** [comment]
Dependend on what you need you might want to check [this](https://github.com/usebruno/bruno/discussions/385#discussioncomment-7813010) as a workaround for e.g. the Client Credentials flow.

**Comment by ykburda** [comment]
Hi,
I have newly started using Bruno. It is very good alternative to postgirl :) We are using oauth2 in projects and i should say if it is fully supported some major authentication methods ,Bruno will be first class app.
